v1/test/cases/testdata/v0/rendertemplate/rendertemplate.yaml

```
@@ -45,5 +45,6 @@ cases:
         template_string = `{{.testvarnotprovided}}`
```

We should probably also have a case like
```suggestion
        template_string = `var: [{{.testvarnotprovided}}]`
```
just to be sure we also deal with the template, not just return a `<no value>` (or `<undefined>`)